packages/trimerge-sync-basic-server/src/server.ts

```diff
@@ -40,8 +42,12 @@ export class BasicServer {
   }
 
   listen(port: number) {
-    const wss = new Server({ port });
-
+    const app = express();
+    app.get('/health', (req, res) => {
+      res.status(200).send('ok');
+    });
+    const server = createServer(app);
+    const wss = new Server({ server });
     wss.on('listening', () => {
       console.log('listening on: %s', wss.address());
```

Comment:
Does this still log out? Or should it be moved to the express server?

Reply:
true, I did change where we log that